apps/dashboard/package.json

```diff
@@ -31,12 +31,15 @@
     "@emotion/styled": "11.11.5",
     "@hookform/resolvers": "^3.6.0",
     "@n8tb1t/use-scroll-position": "^2.0.3",
+    "@radix-ui/react-aspect-ratio": "^1.0.3",
```

Comment:
not a blocker: what does this actually do? I think we can do aspect ratio stuff with pure css/tailwind?

Reply:
This was added because it was recommended in shadcn - https://ui.shadcn.com/docs/components/aspect-ratio